superset/connectors/base/models.py

```diff
@@ -28,6 +28,10 @@
 from superset.utils import core as utils
 
 
+# If the column_name is one of these, is_dttm is set to True
+TIME_COLUMN_NAMES = ['__time']
```

Comment:
@mistercrunch any reason this isn’t part of `db_engine_specs.py`?

Reply:
It's true that it could very well be in there. I thought the risk of collision with non-time related column with name `__time` wasn't a significant risk.

Reply:
@john-bodley  do you think i should make this tiny bit of logic druid-engine specific?